components/prize_pool/commons/prize_pool.lua

```diff
@@ -855,13 +872,83 @@ function PrizePool:_storeData()
 				lpdbEntry
 			)
 		end
+	end
 
-		if self.options.storeSmw then
-			Template.safeExpand(mw.getCurrentFrame(), 'PrizePoolSmwStorage', lpdbEntry)
+	return self
+end
+
+function PrizePool:_storeSmw(lpdbEntry, smwTournamentStash)
+	local smwEntry = self:_lpdbToSmw(lpdbEntry)
+
+	if self._smwInjector then
+		smwEntry = self._smwInjector:adjust(smwEntry, lpdbEntry)
+	end
+
+	local count = tonumber(tournamentVars:get('smwRecords.count')) or 0
+	count = count + 1
+	tournamentVars:set('smwRecords.count', count + 1)
```

Comment:
```suggestion
	local count = (tonumber(tournamentVars:get('smwRecords.count')) or 0) + 1
	tournamentVars:set('smwRecords.count', count)
```

Reply:
(old had double `+ 1` also?)

Reply:
it was a plain copy paste, so yeah it had the double +1
tested it withouth though and works fine

Reply:
Is it needed then?

Reply:
no, hence i kicked it^^